package.json

```diff
@@ -3,14 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.1.2",
```

Comment:
Are we using `material-ui` or `materialize`? 🤔

Reply:
I don't think I understand what you mean. If these `material-ui` package was used in prior chores, Why is it appearing as a new dependency in your PR?

Reply:
Sorry about that. this response below answers this question. https://github.com/andela/ah-frontend-iroquois/pull/11#discussion_r224997623